src/Control/Email/EmailSender.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Control\Email;
+
+use SilverStripe\Core\Injector\Injector;
+use Symfony\Component\Mailer\Messenger\SendEmailMessage;
+use Symfony\Component\Mailer\Transport\TransportInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Transport\Sender\SenderInterface;
+use Symfony\Component\Mime\RawMessage;
+
+class EmailSender implements SenderInterface
+{
+    public function send(Envelope $envelope): Envelope
+    {
+        /** @var SendEmailMessage $sendEmailMessage */
+        $sendEmailMessage = $envelope->getMessage();
+        /** @var RawMessage $rawMessage */
+        $rawMessage = $sendEmailMessage->getMessage();
```

Comment:
```suggestion
        /** @var RawMessage $rawMessage */
        $rawMessage = $envelope->getMessage()->getMessage();
```
We don't need `$sendEmailMessage`